TrackletAlgorithm/MatchEngineUnit_parameters.h

```diff
@@ -52,6 +52,16 @@ static const ap_uint<(1 << (2 * kNBitsBuffer))> emptyUnit() {
   return lut;
 }
 
+template<int kNBits>
+static const ap_uint<(1 << kNBits)> zero() {
+  ap_uint<(1 << kNBits)> lut;
+  for(int i = 0; i < (1 << kNBits); ++i) {
+#pragma HLS unroll
+    lut[i] = (i == 0);
+  }
+  return lut;
+}
+
```

Comment:
@aryd @bryates Can we remove this function, since it isn't used anywhere anymore?